backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +53,16 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+	@PutMapping({ "/{id}" })
+	@Operation(summary = "Update a user by ID")
+	@ApiResponse(responseCode = "204", description = "The request has been successfully processed.")
+	public void updateUserById(
```

Comment:
I don't know if we want to have a PUT method for the aggregate root.. it gives the consumer the ability to accidentally delete the entire entity graph for the user.

I would prefer that we implement PATCH for the user, and then we have more control over what the API consumer can change.

I would remove this method.

Reply:
fixed

Reply:
As implemented, this isn't a JSON PATCH operation. We can fix it in a future PR, but please add a `TODO` so we don't forget.